src/background.ts

```diff
@@ -29,3 +32,10 @@ chrome.runtime.onMessageExternal.addListener((request, sender, sendResponse) =>
     unifiedHandler(request, sender, sendResponse);
     return true;
 });
+
+gStore.get<SettingsType>(StorageKey.SETTINGS).then((settings) => {
```

Comment:
sounds good

Reply:
so actually a feature i would like to keep is the ability to seamlessly add new settings and not break anything: if we add a field to `DEFAULT_SETTINGS` then next time the service worker is loaded it automatically does this get/set thing. if we encapsulate getters/setters then is it okay to be setting every single time we are getting (just in case default settings has extra fields that settings doesnt have)

Reply:
We need to think about whether we want to change the defaults in the future. If you're writing defaults like this, then you're also overwriting the state of whether the user has actually ever  toggled the setting themselves.

Instead, if we just _on the fly_, coerce an `undefined` (which signifies it is unset) into the default value, then we can change defaults in the future.